languages.toml

```diff
@@ -1914,7 +1914,7 @@ source = { git = "https://github.com/sogaiu/tree-sitter-clojure", rev = "e57c569
 [[language]]
 name = "starlark"
 scope = "source.starlark"
-injection-regex = "(starlark|bzl|bazel)"
+injection-regex = "(starlark|star|bzl|bazel)"
 file-types = ["bzl", "bazel", "BUILD"]
```

Comment:
You can add `"star" to the `file-types` list here so that the extension is recognized. `injection-regex` is used for features like markdown codeblocks, so triple backticks and `star` would be a starlark codeblock with this change.

If you have `cargo` set up locally you can try the changes out with `cargo run file.star`